projects/packages/my-jetpack/src/class-wpcom-products.php

```diff
@@ -260,6 +265,9 @@ public static function get_site_current_purchases() {
 
 		$body      = wp_remote_retrieve_body( $response );
 		$purchases = json_decode( $body );
+		// Set a transient for 30 seconds
+		set_transient( self::MY_JETPACK_PURCHASES_TRANSIENT_KEY, $purchases, 30 );
```

Comment:
This looks good, just curious, how have you came up with value of `30 seconds`?

Also, the comment wouldn't be necessary if you would use `MINUTE_IN_SECONDS / 2`.

Reply:
>  just curious, how have you came up with value of 30 seconds?

It's somewhat arbitrary, I wanted something long enough to stick around for the My Jetpack page to load and to handle async requests that needed to fetch products, but not much longer than that so that it would not get stale